ironfish/src/migrations/data/013-wallet-2.ts

```diff
@@ -426,6 +431,24 @@ export class Migration013 extends Migration {
 
     return { unconfirmedBalances }
   }
+
+  private async checkNullifierToNote(stores: Stores, tx: IDatabaseTransaction) {
+    let missing = 0
+
+    for await (const noteHash of stores.new.nullifierToNoteHash.getAllValuesIter(tx)) {
+      const hasNote = await stores.new.decryptedNotes.has(noteHash, tx)
+
+      if (!hasNote) {
+        missing++
+      }
+    }
+
+    if (missing) {
+      throw new Error(
+        `Your wallet is corrupt and missing ${missing} notes for nullifiers. You should delete your accounts database and run this again.`,
```

Comment:
Should we provide any guidance here on how to delete or reset the accounts database?

Reply:
Fixed